x/session/keeper/keeper.go

```diff
@@ -60,12 +60,12 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) BeginBlocker(ctx sdk.Context) {
 	// ctx.BlockHeader().LastBlockId.Hash is the hash of the last block committed
 	hash := ctx.BlockHeader().LastBlockId.Hash
-	// ctx.BlockHeader().Height is the height of the current block being processed,
-	// decrementing it by 1 gives us the height of the last block committed.
-	height := ctx.BlockHeader().Height - 1
-	// If height is 0, ctx.BlockHeader().LastBlockId.Hash will be nil,
-	// fall back to an empty byte slice.
-	if height == 0 {
+	// ctx.BlockHeader().Height is the height of the last committed block.
+	height := ctx.BlockHeader().Height
+	// When height is 1, ctx.BlockHeader().LastBlockId.Hash will not be populated,
```

Comment:
@red-0ne Please create a ticket for `We are also assuming that genesis starts at block 0. We should revisit our session start/end calculations and their corresponding tests. Will create a ticket for that.` and add a TODO_TECHDEBT in the code in the appropriate location.